src/getMongoCachedJSONFetcher.js

```diff
@@ -92,16 +92,19 @@ const getKeyResolver = (cacheConfig, key, url) => {
         let notModified = false;
 
         // find meta only, we don't want to hold potentially large data in memory during the fetch
-        const meta = await findRecord(cacheConfig.collection, key, metaProjection);
+        const meta = {
```

Comment:
I'd compose the logged object in the place of usage, there is no need to bound such information here. The "meta" word here means something different to "meta" data of log.

Also, the URL could be helpful in the logged metadata